tests/CI/exportCSLoop.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# This script will export to the `Production.cfg` file to the
+# yaml format for diracx every 5 seconds
+
+source /home/dirac/ServerInstallDIR/bashrc
+git config --global user.name "DIRAC Server CI"
+git config --global user.email "dirac-server-ci@invalid"
+
+while true;
+do
+    curl -L https://gitlab.cern.ch/chaen/chris-hackaton-cs/-/raw/master/convert-from-legacy.py |DIRAC_COMPAT_ENABLE_CS_CONVERSION=True  ~/ServerInstallDIR/diracos/bin/python - ~/ServerInstallDIR/etc/Production.cfg /cs_store/initialRepo/
```

Comment:
Still something from custom location? Why?

Reply:
Because it is the only place it lives so far. I think eventually it will probably be one export script per VO, but maybe some common tools, but for the time being, this is the best we have.